### PR TITLE
Add cross OpInfo

### PR DESCRIPTION
**Pull request by mruberry**
One of the tasks in https://github.com/pytorch/pytorch/issues/54261.

### CODE REVIEW

**Comment by imaginary-person** [comment]
Lots of `OpInfo`s added today. I'll careful not to duplicate :)

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -2702,6 +2711,17 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
...
+           dtypesIfCPU=all_types_and_complex(),
+           dtypesIfCUDA=all_types_and(torch.half),
+           sample_inputs_func=sample_inputs_cross,
+           supports_complex_autograd=False,
+           skips=(
+               # AssertionError: UserWarning not triggered :
+               # Resized a non-empty tensor but did not warn about it.
+               SkipInfo('TestCommon', 'test_out'),
+           )
+    ),
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:2724]: TODO: make flake happy

**Review by anjali411 — APPROVED** [review]
nice! 

Please remove the autograd tests added in this PR: https://github.com/pytorch/pytorch/pull/55854

**Comment by mrshenli** [comment]
Hey @mruberry pytorch_windows_vs2019_py36_cuda11.1_test1 starts to fail after this commit. Do you know if that failure is relevant to this PR?

https://app.circleci.com/pipelines/github/pytorch/pytorch/314843/workflows/eb9c6b01-c083-4ad4-ac57-d5e4c6ef0295/jobs/13091954/steps

**Comment by imaginary-person** [comment]
@mruberry, it's probably another CUDA 11.1 bug, as [this test had passed with CUDA 10.1](https://app.circleci.com/pipelines/github/pytorch/pytorch/314727/workflows/d57406bf-a3c4-431b-b199-f50e0d8fe50f/jobs/13085778/steps).

**Comment by mruberry** [comment]
> Hey @mruberry pytorch_windows_vs2019_py36_cuda11.1_test1 starts to fail after this commit. Do you know if that failure is relevant to this PR?
> 
> https://app.circleci.com/pipelines/github/pytorch/pytorch/314843/workflows/eb9c6b01-c083-4ad4-ac57-d5e4c6ef0295/jobs/13091954/steps

Definitely related!

**Comment by imaginary-person** [comment]
That bug isn't present in CUDA 11.3, which would be added to CI soon (#56721). Didn't test with CUDA 11.2, though.

**Pull request closed without merging**